apps/authoring/src/renderer/pages/editor/elements/pane-details/elements/glossary/gloassary-list.tsx

```diff
@@ -11,8 +11,16 @@ export const GlossaryListEntries = ({
   const entires = Object.keys(glossary).sort();
   const getEntryIndex = (ev: React.MouseEvent<Element, MouseEvent>) => {
     const target = ev.target as HTMLElement;
-    const actionBtn = target.parentElement?.parentElement
-      ?.parentElement as HTMLElement;
+
```

Comment:
The idea here is to have something that will work for all ActionMenus rather than be specific to the glossary's. Updated the ActionMenu to use spans for the label, and if a user clicks on that part of the item, navigate up one additional parentNode